doc/cephadm/troubleshooting.rst

```diff
@@ -176,3 +176,39 @@ To access the admin socket, first enter the daemon container on the host::
 
     [root@mon1 ~]# cephadm enter --name <daemon-name>
     [ceph: root@mon1 /]# ceph --admin-daemon /var/run/ceph/ceph-<daemon-name>.asok config show
+
+Manually deploying a MGR daemon
+-------------------------------
+
+cephadm requires a MGR daemon in order to manage the cluster. Please
+follow these steps in order to deploy a MGR ``mgr.hostname.smfvfd`` manually.
+
+Disable the cephadm scheduler, in order to prevent cephadm from removing the new 
+MGR::
+
+  ceph config set mgr mgr/cephadm/pause true
```

Comment:
[ceph: root@cephLab2-node-00 /]# ceph config set mgr mgr/cephadm/pause true
Error EINVAL: unrecognized config option 'mgr/cephadm/pause

Maybe if what we want is to prevent the orchestrator start to do things when deployed .. we need to use:
ceph orch pause

Reply:
at this point, there is no MGR that could handle this command

Reply:
Note: If the problem to resolve is to add a mgr daemon in a cluster with only one node, the first thing to do is provide a way to execute ceph commands, the ceph shell does not start properly without 1 mgr.

Reply:
it's more like, someone removed their last MGR in a cluster. If you didn't complete the bootstrap process, you probably want to start form scratch

Reply:
fixed